core/src/main/java/org/ldaptive/dn/DefaultAttributeValueEscaper.java

```diff
@@ -0,0 +1,94 @@
+/* See LICENSE for licensing and NOTICE for copyright. */
+package org.ldaptive.dn;
+
+import org.ldaptive.LdapUtils;
+
+/**
+ * Escapes an attribute value per RFC 4514 section 2.4.
+ *
+ * @author  Middleware Services
+ */
+public class DefaultAttributeValueEscaper implements AttributeValueEscaper
```

Comment:
At first sight this looked out of scope, but I see it's used in `DefaultRDnNormalizer`. I think packaging this functionality in a component is a beneficial change, but putting it in the `dn` package seems wrong. It can obviously stand on its own and is simply an implementation detail of components in this package. Consider moving to a more suitable package.

Reply:
This is essentially the same comment as above, no?

Reply:
Yes, same as packaging above.